### PR TITLE
added usage examples for tf.image

### PR DESCRIPTION
**Pull request by rpalakkal**
added usage examples for all the "random" operations under tf.image api doc

### CODE REVIEW

**Comment by rpalakkal** [comment]
@googlebot I signed it!

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -354,6 +354,14 @@ def random_flip_up_down(image, seed=None):
 
   Returns:
     A tensor of the same type and shape as `image`.
+
```
Comment (rahul2240) [inline tensorflow/python/ops/image_ops_impl.py]: Hey! :smile:  We already have an example for this function.

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -395,6 +403,13 @@ def random_flip_left_right(image, seed=None):
   Returns:
     A tensor of the same type and shape as `image`.
 
+  Usage Example:
```
Comment (rahul2240) [inline tensorflow/python/ops/image_ops_impl.py]: Already an example is present in the function.

File: tensorflow/python/ops/image_ops_impl.py (hunk 1 above)
Comment (rpalakkal) [reply tensorflow/python/ops/image_ops_impl.py]: oh oops my bad, I'll remove that

File: tensorflow/python/ops/image_ops_impl.py (hunk 2 above)
Comment (rpalakkal) [reply tensorflow/python/ops/image_ops_impl.py]: didn't see that - will remove

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -1601,6 +1623,12 @@ def random_contrast(image, lower, upper, seed=None):
   Returns:
     The contrast-adjusted image(s).
 
+  Usage Example:
+    ```python
+    import tensorflow as tf
+    x = tf.random.normal(shape=(256, 256, 3))
+    tf.image.random_contrast(x, 0, 10)
+    ```
```
Comment (rahul2240) [inline tensorflow/python/ops/image_ops_impl.py]: Let's maintain uniformity and leave a line after the comment.

Comment (rahul2240) [inline tensorflow/python/ops/image_ops_impl.py]: Let's maintain uniformity and leave a line after the comment as you have done with the previous examples

Comment (rpalakkal) [reply tensorflow/python/ops/image_ops_impl.py]: Just changed it. I'll make sure to keep that in mind in the future! Thanks for the feedback :)

**Comment by rahul2240** [comment]
Thanks for the PR @rpalakkal :tada: . Don't you think we should have a single commit for this PR? Having 3 commits where 2 commits are just about removing a few lines from the first commit doesn't make any sense. Can you convert this into a single commit? Feel free to ask if you face any problem :smile: .

**Comment by rpalakkal** [comment]
@rahul2240 Totally agree - just squashed the 3 commits into one(and learned about using git rebase!) Thanks

**Review by divyanshu-rawat — DISMISSED** [review]
Thanks for the PR @rpalakkal LGTM 👍

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Please follow indications at https://www.tensorflow.org/community/contribute/docs_ref

**Comment by gbaned** [comment]
@rpalakkal Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**